src/language/walker/syntaxWalker.ts

```diff
@@ -178,6 +178,10 @@ export class SyntaxWalker {
         this.walkChildren(node);
     }
 
+    protected visitJsxSpreadAttribute(node: ts.JsxSpreadAttribute) {
```

Comment:
this change is unrelated; please submit this in a separate PR (we squash commits before merging so it would be confusing to find this in the bugfix commit)